### PR TITLE
Fixed:Implementation of Qr code Dynamically

### PR DESCRIPTION
**Pull request by GOKULGSECE**
Fixed: #138
I have implemented a QR code generator dynamically based upon the user's input hereby attaching the screenshot /videoof the same

https://github.com/oxiton-foundation/click-metrics/assets/125350820/637789a4-8357-4c5d-a7de-449de880a5bd

Thanks for assigning the issue to me

### CODE REVIEW

File: client/src/components/pages/App.css [hunk 1]
```diff
@@ -0,0 +1,99 @@
+/* App.css */
```
Comment (Swarnendu0123) [inline client/src/components/pages/App.css:1]: Don't use inline css. Use tailwind.

**Comment by Swarnendu0123** [comment]
There arenfew suggesions

- By default we can render the qr of click matrics.
- Use buttons in black.
- Decrease the width of the input.
- Buttons can be on the same horizontal level.

File: client/package.json [hunk 2]
```diff
@@ -16,6 +16,7 @@
     "@radix-ui/react-label": "^2.0.2",
     "@radix-ui/react-slot": "^1.0.2",
     "antd": "^5.18.0",
+    "antf": "^1.0.0",
```
Comment (Swarnendu0123) [inline client/package.json:19]: What is the function of this new library?

File: .hintrc [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (Swarnendu0123) [inline .hintrc:1]: Why did you add this file? Is there a specific purpose of this file?

**Comment by Swarnendu0123** [comment]
There arenfew suggesions

- By default we can render the qr of click matrics.
- Use buttons in black.
- Decrease the width of the input.
- Buttons can be on the same horizontal level.
- You should write Fixes #issue_number because if your PR gets merged the issue will be closed, otherwise we have to manually close it.

File: client/package.json (hunk 2 above)
Comment (GOKULGSECE) [reply client/package.json:19]: As we can get a message stating that the QR haa been generated as a popup

File: client/src/components/pages/App.css (hunk 1 above)
Comment (GOKULGSECE) [reply client/src/components/pages/App.css:1]: Ok I will implement tailwind In this project

File: .hintrc (hunk 3 above)
Comment (GOKULGSECE) [reply .hintrc:1]: Actually while committing the changes via a message this file is getting edited

**Comment by GOKULGSECE** [comment]
Oke I will resolve the reviews and raise a PR accordingly

**Comment by GOKULGSECE** [comment]
i have re-checked and requested for the PR as i have implemented Tailwind CSS for the following and i have changed as per your request Thank you for Assigning me this issue

File: client/src/components/pages/qr-codes.tsx [hunk 4]
```diff
@@ -1,31 +1,96 @@
...
-            });
-        }
-    };
+  const generateQR = async () => {
+    try {
+      if (qrinput.trim() === "") {
+        message.error("Input cannot be empty");
+        return;
+      }
+      const url = `https://api.qrserver.com/v1/create-qr-code/?size=150x150&data=${encodeURIComponent(
```
Comment (Swarnendu0123) [inline client/src/components/pages/qr-codes.tsx:15]: What is the purpose of this url?

Comment (GOKULGSECE) [reply client/src/components/pages/qr-codes.tsx:15]: It helps to generate the QRcode dynamically ,using this url it can be able to generate QR code with respect to the input given by the user. Basically it is like a API for generating QR code generation

**Comment by Swarnendu0123** [comment]
Can you please attach one video after the final changes you have done?

**Comment by GOKULGSECE** [comment]
HEreby i am attaching the video for my implementation

https://github.com/oxiton-foundation/click-metrics/assets/125350820/8aa7cd11-a362-4cb0-ad22-0bfe0340451e

**Comment by Swarnendu0123** [comment]
Thabks:) Great Work!
